### ISSUE TITLE
Sharing: do not use !important to allow plugin and theme developers to overwrite default styles

### ISSUE THREAD

**Issue by jeherve** [Automattic/jetpack#1922]
Jetpack's Sharing module includes quite a few `!important` in its CSS:
https://github.com/Automattic/jetpack/blob/master/modules/sharedaddy/sharing.css

We should avoid doing that if possible to allow plugin and theme developers to overwrite default styles.

Reported in #1500.

**Comment by kraftbj** [comment]
Also requested in https://wordpress.org/support/topic/notification-by-mail?replies=3

**Comment by jeffgolenski** [comment]
Agreed. Still a lot of code auditing I need to do. Added these to my list for 3.8!
